docs-web/configuration/transportserver-resource.md

```diff
@@ -391,6 +391,27 @@ spec:
     port: 80
 ```
 
+Snippets can also be specified for a stream. In the example below, we use snippets to configure [access control](http://nginx.org/en/docs/stream/ngx_stream_access_module.html) and we set the [logging level](http://nginx.org/en/docs/http/ngx_http_limit_conn_module.html#limit_conn_zone) for cases when the server limits the number of connections in a TransportServer:
```

Comment:
better to configure a complete feature, rather than just a parameter. otherwise, we configure a parameter for a feature that is not enabled and configured.

(I also fixed the link - it was for http module, not for stream)

```suggestion
Snippets can also be specified for a stream. In the example below, we use snippets to configure [access control](http://nginx.org/en/docs/stream/ngx_stream_access_module.html) and [limit the number of connections](https://nginx.org/en/docs/stream/ngx_stream_limit_conn_module.html):
```